src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Ink/InkSerializedFormat/DrawingAttributeSerializer.cs

```diff
@@ -189,7 +189,22 @@ internal static uint DecodeAsISF(Stream stream, GuidList guidList, uint maximumS
                     }
 
                     byte[] data = new byte[ropSize];
-                    stream.Read (data, 0, (int)ropSize);
+#if NET7_0_OR_GREATER
```

Comment:
Why did you add `#if` ? This project should always build with NET7_0_OR_GREATER set to true so I don't understand the value of having the `#else`.

Reply:
I'm curious as well... doesn't this only build for net9.0?
https://github.com/dotnet/wpf/blob/47efe4600a00ac059aeca109459e9a4e8132d907/Directory.Build.props#L5-L6

Reply:
+1, we will remove it. Our first try was to just disable it, then we checked the runtime PR and added same changes to call `ReadExactly`

Reply:
+1, We will remove it. Our first try was to disable it, then we checked on the runtime changes and added the same changes.